Point.h

```diff
@@ -0,0 +1,29 @@
+#ifndef POINT_H
+#define POINT_H
+
+ /* ----------------------------------------------------------------------------
+   * Copyright &copy; 2015 Randy Tanizawa <rtanizawa@csu.fullerton.edu>
+   * Released under the [MIT License] (http://opensource.org/licenses/MIT)
+   * ------------------------------------------------------------------------- */
+
+  /**
+   * Describing a class called Point.
+   */
+
+class Point
+{
+    private:
+        double x;
+        double y;
+    public:
+        Point();
+        Point(double X, double Y);
+        double getPoint_x() const;
+        double getPoint_y() const;
+        double distance(const Point& a) const;
+        void move_P(double X, double Y){ x = X; y = Y;};
```

Comment:
just from the function name, it would be unclear to me whether this was moving the point to an absolute position, or shifting the point by (x,y).  also, it should probably be implemented in the .cpp file, with everything else :)